.github/workflows/build-test-deploy.yaml

```diff
@@ -233,10 +233,10 @@ jobs:
         with:
           go-version: "1.23"
 
-      - uses: sigstore/cosign-installer@v3
+      - uses: sigstore/cosign-installer@v3.8.1
         with:
           # DO NOT USE v2 until we decide on whether to use Rekor or not
-          cosign-release: "v1.13.1"  # Binary version to install
+          cosign-release: "v2.4.3"  # Binary version to install
```

Comment:
Could we remove this now and use the latest but continue to pin the action version?

Reply:
I would prefer to have this version pinned because it makes the builds predictable and stable (e.g. we are safe from a bad cosign-release version that got pushed). Why do you want to use latest?